frontend/app/src/StreamlitLib.test.tsx

```diff
@@ -35,17 +36,16 @@ import {
   Delta as DeltaProto,
   Element as ElementProto,
   ForwardMsgMetadata as ForwardMsgMetadataProto,
-  IAppPage,
   Text as TextProto,
-  WidgetStates,
   TextElement,
+  render,
 } from "@streamlit/lib"
```

Comment:
Nit: My VSCode showing the following imports from "@streamlit/lib" no longer used & can be removed:
* AlertElement
* mount
* TextElement

Reply:
Oops thanks. I'm surprised the linter did not catch this. I think I need to make a PR to have it catch for the app side of things if I had to guess...